From 46ae828be9dfb0056184fb9f682ae06e74acc545 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Wed, 16 May 2018 08:59:32 +0800
Subject: [PATCH] clk: rockchip: rk3288: mark the aclk_dmac1 as critical clk

crypto and dmac share the same noc clk,
so mark the aclk_dmac1 as critical clk.

Change-Id: I34a4a7cc532a385086679fafb961a47b0a6abc3b
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk3288.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/rockchip/clk-rk3288.c b/drivers/clk/rockchip/clk-rk3288.c
index f82084fefc50..a681f3c51f93 100644
--- a/drivers/clk/rockchip/clk-rk3288.c
+++ b/drivers/clk/rockchip/clk-rk3288.c
@@ -829,6 +829,7 @@ static const char *const rk3288_critical_clocks[] __initconst = {
 	"pclk_ddrupctl1",
 	"pclk_publ1",
 	"pmu_hclk_otg0",
+	"aclk_dmac1",
 };
 
 static void __iomem *rk3288_cru_base;
-- 
2.35.3

